account_financial_report_qweb/report/abstract_report.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models
+
+
+class AbstractReport(models.AbstractModel):
+    _name = 'report_qweb_abstract'
+
+    def _transient_clean_rows_older_than(self, cr, seconds):
+        assert self._transient, \
+            "Model %s is not transient, it cannot be vacuumed!" % self._name
+        # Never delete rows used in last 5 minutes
+        seconds = max(seconds, 300)
+        query = """
+DELETE FROM """ + self._table + """
```

Comment:
Did you tested how much time it takes for deleting 80 millions of rows?

Reply:
No much as I expected actually:

```
delete from report_general_ledger_qweb_move_line;
DELETE 86000001
Time: 74372.450 ms (01:14.372)
```
Admittedly my query does not include the where clause but at worse it adds a few seconds.
A couple of minutes for the vacuum is probably acceptable.